geonode/harvesting/migrations/0028_auto_20210730_0438.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.4 on 2021-07-30 04:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('harvesting', '0027_remove_harvester_default_access_permissions'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='harvester',
+            name='num_harvestable_resources',
+            field=models.IntegerField(blank=True, null=True),
+        ),
+    ]
```

Comment:
Please regenerate this migration file, since there is now already a more recent migration starting with `0028_`

Reply:
updated @ricardogsilva